File: TCS_NQT_Coding_Problems/Leap_Year.cpp
#include<bits/stdc++.h>
using namespace std;

bool check_year(int y){
    if((y % 400 == 0) || ((y % 100 != 0) && (y % 4 == 0)))
        return true;
    return false;
}

int main(){
    int year;
    cout << "Enter Year :- " << " ";
    cin >> year;
    check_year(year)? 
        cout << "It is a Leap Year" << endl :
        cout << "It is not a Leap Year" << endl;
    return 0;
}

File: TCS_NQT_Coding_Problems/Leap_Year.py
def check_year(year):

    if year % 400 == 0 or (year % 100 != 0 and year % 4 == 0):
        return True
    else:
        return  False


_year = int(input())
print("It is a Leap Year") if check_year(_year) else print("It is not a Leap Year")
